A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -2102,6 +2155,42 @@
                         <ONLY_SEQ1/>
                     </UseTemplate>
 
+                    <!-- Data Loading Selector-->
+                    <Component ID = "Data_Loading_Selector">
```

Comment:
Why do some of the attributes you added have a space around the =? Please follow the rest of the file and normal XML formatting

Reply:
There are some more occurences of this

Reply:
Fixed the other 5 or so occurrences.